session_2/student_registry/app/models/student.rb

```diff
@@ -0,0 +1,9 @@
+class Student < ApplicationRecord
+  validates_presence_of :name, :roll_number, :branch, :address, :admission_year, :cgpa, message: 'cannot be null',
```

Comment:
`validates_presence_of` is the old syntax.

Prefer using `validates :name, :roll_number, ..., presence: true`

Reply:
alright, noted! =))